src/Homepage/Homepage.scss

```diff
@@ -4,7 +4,7 @@
 .main-title{
   font-size: 4em;
   letter-spacing: 3px;
-  color: #931a25;
+  color: #7ea04d;
```

Comment:
Note: Just to remind us to come back and change the colors to variables in scss later